ports/mapbox-geojson-vt-cpp/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "mapbox-geojson-vt-cpp",
-  "version-semver": "6.6.4",
+  "version-semver": "6.6.5",
```

```suggestion
  "version": "6.6.5",
```
Seems there is no any evidence that can describe this meets the policy of semver.

---

I did find this mention of semver by what appears to be one of the maintainers: https://github.com/mapbox/geojson-vt-cpp/pull/21#issue-120041154

---

Thanks for your information.